libsnark/gadgetlib1/gadgets/hashes/anemoi/tests/test_anemoi_gadget.cpp

```diff
@@ -339,31 +350,115 @@ void test_anemoi_permutation_mds()
     libff::print_time("anemoi_permutation_mds tests successful");
 }
 
-template<typename ppT> void test_for_curve()
+template<typename ppT> void test_intermediate_gadgets_bls12_381()
```

Comment:
Why is this function named `...bls12_381` but still has a `ppT` type param?

Reply:
Fixed by removing template specialization.